.pre-commit-config.yaml

```diff
@@ -144,6 +144,37 @@ repos:
         language: system
         types: [python]
 
+  - repo: local
```

Comment:
Bazel team and copyright are two major differentiation I personally suffered between CI and precommit hook / local formatting script.